config-model/src/test/java/com/yahoo/vespa/model/application/validation/change/ContentTypeRemovalValidatorTest.java

```diff
@@ -40,8 +41,16 @@ public void testContentTypeRemovalValidation() {
     public void testOverridingContentTypeRemovalValidation() {
         ValidationTester tester = new ValidationTester();
 
-        VespaModel previous = tester.deploy(null, getServices("music"), null).getFirst();
-        tester.deploy(previous, getServices("book"), removalOverride); // Allowed due to override
+        VespaModel previous = tester.deploy(null, getServices("music"), Environment.prod, null).getFirst();
+        tester.deploy(previous, getServices("book"), Environment.prod, removalOverride); // Allowed due to override
+    }
+
+    @Test
+    public void testNoOverrideNeededinDev() {
+        ValidationTester tester = new ValidationTester();
+
+        VespaModel previous = tester.deploy(null, getServices("music"), Environment.prod, null).getFirst();
+        tester.deploy(previous, getServices("book"), Environment.dev, null);
```

Comment:
Will this always throw if something with the deployment is not allowed? Should we also check that the returned list of config change actions is empty?

Reply:
Yes, it would throw if not allowed (see the first test in the class, which does the same but for prod). I don't see why we should check config change actions. This has no impact on them one way or the other.

Reply:
Ok 👍